docs/write_your_tiltfile.md

```diff
@@ -18,6 +18,9 @@ k8s_yaml(['foo.yaml', 'bar.yaml'])
 # run kustomize to generate yaml
 k8s_yaml(kustomize('config_dir'))
 
+# run helm to generate yaml
+k8s_yaml(helm('chart_dir'))
+
 # run a custom command to generate yaml
```

Comment:
Can we move local (the general case) above the specific cases (helm and kustomize), and then maybe a comment like "we also handle common tools to generate yaml"?